### ISSUE TITLE
a11y | tab order

### ISSUE THREAD

**Issue by scottqueen-bixal** [GSA/px-benefit-finder#188]
11 | 08/31 | Amy | “Continue” button skipped in keyboard order. | Anyone using a keyboard only or filling out the form without a mouse should be able to complete all actions in the form. | I am unable to use my keyboard to select “continue”. Focus skips from “back” to “top” and there isn’t any way to “continue” using my keyboard. WCAG 2.1.1 | Open the “about the deceased” page. Without using a mouse, tab through each element and fill out the form. Then try to click “continue”. Used Edge via VDI. | “About the deceased” page.See screen shot. | Development | Blank |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

12 | 08/31 | Amy | Modal window is  not keyboard accessible | Anyone using a keyboard only or filling out the form without a mouse should be able to complete all actions in the form. | Using a keyboard only, I cannot select either the “review selections” or the “see results”. I CAN escape out of the modal so technically it’s not a keyboard trap. But I cannot perform the functions to finalize the form.  WCAG 2.1.1 | If you click “continue” with your mouse (since you can’t use a keyboard) you are taken to the modal window. Try using your keyboard to tab between the close button (which receives automatic focus) and either of the two buttons. Used Edge via VDI. | Modal “review selections” window.See screen shot. | Development | Blank | Scott to review all a11y items to check if critical, for non critical items will review with Cindy
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

13 | 9/1 | Angelina | “Skip to main content” not able to be activated with keyboard | A user should be able to activate ‘Skip to the main content’ link with keyboard and be taken to the main content without having to listen to all the site navigation. | Unable to activate the “Skip to main content” link with the standard Enter key. WCAG 2.1.1When pressing Space bar, focus was taken to the main content, however, when you hit Tab, it takes you up to the top ‘An official website of the United States government’ “Here’s how you know” element. | Use the keyboard ‘tab’ key to navigate to the “Skip to main content” link. Pressing “Enter” should take your focus to the main content on the page. From there you should be able to press ‘tab’ to go to the next interactive, focusable element within the main page.Used Edge through VDI. | Occurs on all pages | Development | Blank |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --

16 | 9/1 | Angelina | Radio buttons on forms are not correctly formatted and normal keyboard navigation is not expected. | When you tab key through, each radio group gets its own tab, not each button choice. Once the group receives focus, you should be able to press ‘Space’ key to make your selection and use arrow keys to move focus to next button within the group.Screen reader should announce “[No not checked] 1 of 2”W3 Radio Group Pattern guidanceDeque Radio and Radio button example and code | Each button choice is receiving tabbable focus, rather than each radio group.Unable to use arrow keys to change choice selection.ANDI ouput is incorrect. | Occurs on all radio groups using Tab and arrow keys.ANDI was used to view ANDI output.Edge through VDI. | See screenshot item GSA/px-benefit-finder#568 for ANDI output. | Development | Blank |  
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
